src/main/res/values/strings.xml

```diff
@@ -88,20 +88,20 @@
   <string name="new_updates_available">New updates available</string>
   <plurals name="new_updates_description_format">
     <item quantity="one">%d new update.</item>
-    <item quantity="other">%d new updates.</item>
+    <item quantity="other">%d applications have newer versions available.</item>
```

Comment:
This text is displayed in collapsed notification and should be short. Like "%d new update." (which you didn't change for some reason), but "%d new updates.". Please, revert this.